frcnn_detector.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, model_path,
         self.__use_selective_search_rois = use_selective_search_rois
         self.__use_grid_rois = use_grid_rois
         self.__model = None
+        self.__isPythonModel = None

```

Comment:
In general for python I use (or try my best to use) PEP8 notation, which means lower case variable names and underscores as seperators. So __isPythonModel should be changed to __is_python_model